MatchingGame.js

```diff
@@ -2,12 +2,10 @@ const section = document.querySelector("section");
 const playerLivesCount = document.querySelector("span");
 let playerLives = 10;
 
-
```

Comment:
In general double spaces are looked down upon, just use 1 space as a separator. Very minor